### PR TITLE
Add string .IfLengthEquals(length) .IfLengthNotEquals(length)

### LINKED ISSUE #9: Add string .IfLengthEquals(length) .IfLengthNotEquals(length)
Add throwing an exception if the string length equals/doesn't equal the given length

Something like this:

Equals:
```csharp
"str".Throw().IfLengthEquals(3)
```

Not Equals:
```csharp
"str".Throw().IfLengthNotEquals(3)
```

and the corresponding extension methods for properties
```csharp
user.Throw().IfLengthEquals(user => user.Name, 3)
``` 
```csharp
user.Throw().IfLengthNotEquals(user => user.Name, 3)
```

### PR DESCRIPTION
**Pull request by DevJonie**
Resolves #9

### CODE REVIEW

File: src/Validators/Validator.Strings.cs [hunk 1]
```diff
@@ -73,4 +73,22 @@ internal static void ThrowIfNotEquals(string value, string paramName, ExceptionC
             ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String should be equal to '{otherString}'.");
         }
     }
+
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    internal static void IfLengthEquals(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, int length)
```
Comment (mantinband) [inline src/Validators/Validator.Strings.cs:78]:
```suggestion
    internal static void ThrowIfLengthEquals(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, int length)
```
H&B

**Review by mantinband — APPROVED** [review]

**Comment by mantinband** [comment]
Looks great. Thanks for implementing this! You can bump the version from `1.1.0` to `1.1.1` in Throw.csproj so that the changes will deploy to nuget.org

File: src/Validators/Validator.Strings.cs (hunk 1 above)
Comment (DevJonie) [reply src/Validators/Validator.Strings.cs:78]: 😅 missed that convention there. Will fix it

**Comment by mantinband** [comment]
Thanks again @DevJonie 🙌🏽
Would love to hear your feedback and ideas for future features!

**Pull request merged**